LICENSE.md

```diff
@@ -1,89 +1,90 @@
-# Eclipse Public License - v 1.0
-
-    THE ACCOMPANYING PROGRAM IS PROVIDED UNDER THE TERMS OF THIS ECLIPSE PUBLIC LICENSE ("AGREEMENT"). ANY USE, REPRODUCTION OR DISTRIBUTION OF THE PROGRAM CONSTITUTES RECIPIENT’S ACCEPTANCE OF THIS AGREEMENT.
+# Eclipse Public License - v 2.0
```

Comment:
why changing to EPL v2? I'm not sure if we can do this without some extra work first. EPLv1 and EPLv2 are two different licenses afaik (i.e. a project cannot necessarily automatically upgrade from EPLv1 to v2)